deploy/kubernetes/README.md

```diff
@@ -24,6 +24,36 @@ To apply the daemonset to your cluster with example patches applied:
 kustomize build deploy/kubernetes/overlays/examples | kubectl apply -f -
 ```
 
+### cAdvisor with perf support on Kubernetes
+
+Example of modifications needed to deploy cAdvisor with perf support is provided in [overlays/examples_perf](overlays/examples_perf) directory (modification to daemonset and configmap with perf events configuration).
+
+To test example deployment from the cadvisor/ directory build binary with perf support
+```
+GO_FLAGS="-tags=libpfm,netgo" make build
```

Comment:
We don't need users to rebuild cadvisor if the release version has perf support, right?

Reply:
Yes, you're right;)